src/WasmDis.ts

```diff
@@ -806,6 +818,7 @@ export class WasmDisassembler {
   public disassembleChunk(reader: BinaryReader, offsetInModule: number = 0): boolean {
     if (this._done)
       throw new Error('Invalid state: disassembly process was already finished.')
+    let foundCodeSection: boolean = false;
```

Comment:
Generally we don't use function local state in this way. In theory this class can be used as a streaming reader given chunks of data, so the beginning and end of the code section might not be seen in the same call.